salt/hg/config/svn.apache.conf.jinja

```diff
@@ -44,6 +44,13 @@
         Require valid-user
     </Location>
 
+  <IfModule mod_ratelimit.c>
+    <Location /srv/hg/>
```

Comment:
Similarly to the other config, I'm not sure that the hg server would be impacted by this change, just the svn server.